src/Components/Components/src/Routing/TemplateParser.cs

```diff
@@ -18,8 +20,7 @@ namespace Microsoft.AspNetCore.Components.Routing;
 // * Catch-all parameters (Like /blog/{*slug})
 internal sealed class TemplateParser
 {
-    public static readonly char[] InvalidParameterNameCharacters =
-        new char[] { '{', '}', '=', '.' };
+    private static readonly IndexOfAnyValues<char> _invalidParameterNameCharacters = IndexOfAnyValues.Create("{}=.");
```

Comment:
Nit: There seems to be inconsistency in this PR as to whether these static readonly fields use "_underscoreCamelCasing" or "PascalCasing".

Reply:
I tried to follow the conversations in this repo / files. They are different to the ones in runtime.